packages/manager/test/SliceMachineManager-versions-getAllStableAdapterVersions.test.ts

```diff
@@ -0,0 +1,43 @@
+import { expect, it } from "vitest";
+
+import { createTestPlugin } from "./__testutils__/createTestPlugin";
+import { createTestProject } from "./__testutils__/createTestProject";
+import { mockNPMRegistryAPI } from "./__testutils__/mockNPMRegistryAPI";
+
+import { createSliceMachineManager } from "../src";
+
+it("returns stable Adapter versions", async (ctx) => {
```

Comment:
Super tiny nitpick: "adapter" should be lowercase

Reply:
Done :)